### ISSUE TITLE
@/ imports - consider alternatives?

### ISSUE THREAD

**Issue by romeerez** [alan2207/bulletproof-react#55]
Thank you for this amazing source of practices, I've been sharing it with lot of people!

`@/` imports confuses me, could you please add alternatives to the doc? I'm not sure if and why this is a best way.

Another option is to set `baseUrl: 'src'` to tsconfig and now instead of '@/components/X' we import from 'components/X'. I don't think there are meaningful and maintained 'components', 'hooks', 'lib', 'assets' packages on npm.

Another option is to use relative paths, editors are smart nowadays, they write imports automatically, they update imports automatically when moving files. At least VSCode and JetBrains editors a capable. Most of people are using VSCode, so this shouldn't be a problem.

**Comment by baymax-codecare** [comment]
You can consider `~/` instead of `@/`.

**Comment by romeerez** [comment]
it's just the same with a different symbol, while I was suggesting to mention that `baseUrl` exists and seems to be a valid option, and either relative imports is not a big deal with modern editors

**Comment by alan2207** [comment]
Hi there,

Both of your suggestions are valid. I wrote a bit [here](https://github.com/alan2207/bulletproof-react/blob/master/docs/project-configuration.md#absolute-imports
) about the reason why I am choosing this way:

"It is also possible to define multiple paths for various folders(such as @components, @hooks, etc.), but using @/* works very well because it is short enough so there is no need to configure multiple paths and it differs from other dependency modules so there is no confusion in what comes from node_modules and what is our source folder. That means that anything in the src folder can be accessed via @, e.g some file that lives in src/components/MyComponent can be accessed using @/components/MyComponents."

In a nutshell, I find it useful to have a symbol in front of it because it will prevent conflicts from `node_modules` imports in case a folder in `src` has the same name as some dependency  - happens not very often, but it's nice to be sure and not to think about it ever again. In the end, it's just a matter of preference :)

**Comment by romeerez** [comment]
I understand why better to have zero chance than close to zero, and if you're sure @/ is a better practice let's close this issue

**Comment by alan2207** [comment]
It works well with VSCode, you should definitely give it a shot.

And if you have the `import/order` [enabled in eslint](https://github.com/alan2207/bulletproof-react/blob/ff8308ffe773782937e96a1883185eccebfd931e/.eslintrc.js#L48) you will get a nice visual separation of local vs dependency modules which you do not get with the suggested approach.

<img width="1180" alt="Screenshot 2022-01-11 at 09 32 30" src="https://user-images.githubusercontent.com/12713315/148908869-3c19a91b-4ce9-4102-a913-cbb60bfb8c12.png">

 (ignore the squiggly line in the left image for now), VSCode did linting auto fix on save automatically and reordered the imports.
